### PR TITLE
fix: mobile interceptor through media-query instead of MobileView and…

### LINKED ISSUE #193: Fix Mobile Interceptor
- [ ] fix font family (it's currently unset)
- [ ] serve mobile interception view on devices/windows with `width < 1350px` using media queries.

### PR DESCRIPTION
**Pull request by florian-sabonchi**
closes #193

### CODE REVIEW

File: tasks/pages/_app.tsx [hunk 1]
```diff
@@ -22,29 +21,45 @@ const spaceGrotesk = SpaceGrotesk({
...
+          </Head>
+          <QueryClientProvider client={queryClient}>
+            <div className={tw('font-sans')} id="modal-root">
+              <Component {...pageProps} />
+            </div>
+          </QueryClientProvider>
+        </div>
+        <div className="mobile-wrapper">
+          <MobileInterceptor {...pageProps} />
+        </div>S
```
Comment (use-to) [inline tasks/pages/_app.tsx:60]:
```suggestion
        </div>
```

File: tasks/pages/_app.tsx [hunk 2]
```diff
@@ -22,29 +21,45 @@ const spaceGrotesk = SpaceGrotesk({
...
 function MyApp({ Component, pageProps }: AppProps) {
   return (
-    <ProvideLanguage>
-      <BrowserView>
-        <Head>
-          <title>{titleWrapper()}</title>
-          <style>{`
+    <div className={tw(globalStyles)}>
+      <ProvideLanguage>
+        <div className="desktop-wrapper">
```
Comment (use-to) [inline tasks/pages/_app.tsx:42]: use ids

File: tasks/pages/_app.tsx [hunk 3]
```diff
@@ -22,29 +21,45 @@ const spaceGrotesk = SpaceGrotesk({
...
+    .desktop-wrapper{
+      display: none;
+    }
+  }
+
+  @media (min-width: 1350px) {
+    .mobile-wrapper{
+      display: none;
+    }
+  }
```
Comment (use-to) [inline tasks/pages/_app.tsx:35]: use ids instead of classes and parameterize the `1350px` into a variable.

Comment (use-to) [inline tasks/pages/_app.tsx:35]: use ids instead of classes and parameterize the `1350px` into a variable.

Comment (use-to) [inline tasks/pages/_app.tsx:35]: use ids instead of classes and parameterize the `1350px` into a variable.

**Review by use-to — CHANGES_REQUESTED** [review]

File: tasks/pages/_app.tsx (hunk 3 above)
Comment (florian-sabonchi) [reply tasks/pages/_app.tsx:35]: Variables cannot be used in combination with media queries https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries

Comment (use-to) [reply tasks/pages/_app.tsx]: Can't you just move that variable to ts?

**Review by use-to — CHANGES_REQUESTED** [review]

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Please do some clean-up as well
- [ ] Remove `react-device-react` package
- [ ] Remove `tasks/components/MobileInterceptor.tsx`
- [ ] Consider global styles, inline `tw`

**Comment by florian-sabonchi** [comment]
> * MobileInterceptor

MobileInterceptor is still used? or what do you mean exactly?

**Review by MaxSchaefer — DISMISSED** [review]
Good job and research 💪 I really like this approach, looks clean

**Comment by MaxSchaefer** [comment]
I've removed `react-device-detect`

**Review by MaxSchaefer — APPROVED** [review]

**Comment by MaxSchaefer** [comment]
@use-to you requested changes.

**Comment by MaxSchaefer** [comment]
@florian-sabonchi when approved, please squash merge. Thank you!

**Review by shivadharmi — APPROVED** [review]
@florian-sabonchi and @MaxSchaefer Look good to me. I think this might also close another issue #203. Let me check and confirm with you.

**Comment by florian-sabonchi** [comment]
@use-to can you look in it and approve if everything is fine so I can merge

**Review by use-to — APPROVED** [review]

**Pull request merged**